### DISCUSSION TITLE
生产使用canal-adapter全量+增量同步直接从MySQL到Doris这种可行吗

### DISCUSSION THREAD

**Discussion by jacentsao** [apache/doris#11108] [A - General / Q&A]
根据官方文档quickstart配置1fe 1be，每次全量到300条数据就报错了，主要是下面的两种错误
```
 etl failed detailMessage = tablet writer write failed, tablet_id=11004, txn_id=2828, err=-235
success replica num 0 is less than quorum replica num 1 while error backends 
```
如题：
1 . 不知道是否可行，如果可行的话有没有最佳配置推荐了？
2.  如果不行的话，从MySQL全量+增量到Doris官方推荐的方案是什么了？ 
3.  使用Flink CDC + Flink Doris Connector的话，有什么简便的方案把原来大量的MySQL 建表的sql转换为flink和doris的sql了

不胜感激，不知道有没有类似的最佳实践

**Comment by jacentsao** [comment]
看到了官方文档不推荐使用jdbc方式导入Doris

**Reply by dataalive** [reply]
嗯，FAQ里有写不推荐JDBC方式目前，建议是微批的方式。方案的话推荐是Flink CDC。这块可以加下wechat：jnsimba 确认下详细方案。
